src/index.html

```diff
@@ -217,9 +217,41 @@ <h2 class="h2">Next event</h2>
       </svg>
     </section>
 
+    <section class="my-16 tablet:mx-4 px-4">
+      <div class="tablet:w-[720px] desktop:w-[824px] flex flex-col items-center mx-auto tablet:grid grid-cols-2 tablet:grid-rows-smaller desktop:grid-rows-larger tablet:gap-x-4 desktop:gap-x-6">
+        <h2 class="h2 mobile:mb-0 tablet:mb-2.5 tablet:self-end w-[300px] text-center tablet:text-left">Chat with fellow attendees</h2>
```

Comment:
I think we could round `mb-2.5` to just `mb-3`? We haven't used half-Tailwind units anywhere else, so might be nice for consistency?

Also would it be good if we gave the `h2` class Tailwind's `leading-tight` or `leading-snug`, so that the heading line-height is a little less than the default 1.5? It seems like in the designs this one has a line-height of `1`, but I don't think we should go that far.